cachetory/serializers/json.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+import json
+from typing import Generic
+
+from cachetory.interfaces.serializers import Serializer, ValueT
+
+
+class JsonSerializer(Serializer[ValueT, bytes], Generic[ValueT]):
+
+    @classmethod
+    def from_url(cls, url: str) -> JsonSerializer[ValueT]:
+        return JsonSerializer[ValueT]()
+
+    def serialize(self, value: ValueT) -> ValueT:
```

Comment:
Mypy fails here because it expects a serialized data type to be returned (`str` in this case)